python/setup.py

```diff
@@ -27,7 +27,7 @@
 # before these files have been created, so we have to move the files
 # manually.
 
-SUPPORTED_PYTHONS = [(3, 6), (3, 7), (3, 8)]
+SUPPORTED_PYTHONS = [(3, 6), (3, 7), (3, 8), (3, 9)]
```

Comment:
Maybe also add version [classifiers](https://pypi.org/classifiers/) in `setuptools.setup`? Something like:

```python
setuptools.setup(
    ....
    classifiers=[
        ...
        "Programming Language :: Python :: 3.6",
        "Programming Language :: Python :: 3.7",
        "Programming Language :: Python :: 3.8",
        "Programming Language :: Python :: 3.9",
    ]
)
```

There's for example https://pyreadiness.org/3.9/ which uses the classifiers to determine whether a package support specific versions.